department-of-veterans-affairs/next-build
# QA Fixes :  Event Layout

## Description
Relates to #[18494](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18494). (or closes?)


## Testing done

- [x] yarn test -- -u
- [x] yarn test:format
- [x] yarn test:coverage

## Screenshots
Prod
<img width="997" alt="Screenshot 2024-07-15 at 3 41 03 PM" src="https://github.com/user-attachments/assets/6d1c06a9-64ee-495d-84ae-6674516d8e25">
Local
<img width="997" alt="Screenshot 2024-07-15 at 3 41 03 PM" src="https://github.com/user-attachments/assets/0c07cb9c-c77e-4cfb-8aa5-8fc3d5561ed7">

## QA steps
What variations of circumstances  need to be checked?
So Production shows the Register option, while Local says the event has passed. From the information from the article including the dropdown "View other times for this events," local is accurate.
However, when the user clicks the "This is an online event" link, you will see that the event is still on-going, but the newer times are not updated in the article. This would indicate Production is right. 
I need a second opinion on which version should we follow? Should our local version be correct since the information from the the article indicates there are no other events passed May or should we figure out why aren't the other events from eventbrite updating to this article?

```[tasklist]
```
